src/components/PageWrapper/PageWrapper.module.css

```diff
@@ -0,0 +1,4 @@
+.background {
```

Comment:
This is a suggestion for dealing with dynamically changing backgrounds, but I'm not sure it's the best one.
I see that inbox manually sets the same background color on each component instead and to me that seems a bit tedious, but idk - what do you think?

Reply:
This looks like a good approach.
I agree that using CSS variables here is a solid option. It helps centralize style definitions, and its easy to replace later if we have to!